src/api/write.rs

```diff
@@ -29,12 +31,31 @@ impl Client {
         bucket: &str,
         body: impl Into<Body> + Send,
         precision: TimestampPrecision,
+    ) -> Result<(), RequestError> {
+        self.write_line_protocol_with_precision_headers(
+            org,
+            bucket,
+            body,
+            precision,
+            HeaderMap::new(),
+        )
+        .await
+    }
+
+    async fn write_line_protocol_with_precision_headers(
```

Comment:
Kept this private to limit concessions made to backwards compatibility in the future: this way it can be changed without exposing the downstream to those